modules/monitoring-querying-metrics-for-all-projects-as-an-administrator.adoc

```diff
@@ -0,0 +1,47 @@
+:_module-type: PROCEDURE
+// Module included in the following assemblies:
+//
+// * assemblies/managing-metrics.adoc
+
+[id="querying-metrics-for-all-projects-as-an-administrator_{context}"]
+= Querying metrics for all projects as a cluster administrator
+
+[role="_abstract"]
+As a cluster administrator or as a user with view permissions for all projects, you can access metrics for all default {product-title} and user-defined projects in the Metrics UI.
+
+[NOTE]
+====
+Only cluster administrators have access to the third-party UIs provided with {product-title} Monitoring.
+====
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role or with view permissions for all projects.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. In the *Administrator* perspective within the {product-title} web console, select *Monitoring* -> *Metrics*.
```

Comment:
`dedicated-admins` shouldn't have this view (I think)

Reply:
Done. And is it the OpenShift console? Because if it is, I need to hardcode that because {product-title} will reflect only OSD or ROSA.

Reply:
They should have access to the Developer OpenShift console only, not the Administrator OpenShift console

Reply:
Thank you. Done.